libnavigation-core/src/test/java/com/mapbox/navigation/trip/service/MapboxTripServiceTest.kt

```diff
@@ -13,12 +15,25 @@ import org.junit.Test
 class MapboxTripServiceTest {
 
     private lateinit var service: MapboxTripService
-    private val notification: TripNotification = mockk()
+    private val tripNotification: TripNotification = mockk()
+    private val notification: Notification = mockk()
     private val callback: () -> Unit = { }
 
     @Before
     fun setUp() {
-        service = MapboxTripService(notification, callback)
+        service = MapboxTripService(tripNotification, callback)
+        every { tripNotification.getNotificationId() } answers { 1234 }
+        every { tripNotification.getNotification() } answers { notification }
+        every { tripNotification.onTripSessionStopped() } answers { Unit }
```

Comment:
NIT: all unit functions responses can be mocked by changing the declaration to `private val tripNotification: TripNotification = mockk(relaxUnitFun = true)`

Reply:
Implementing the NIT results in **io.mockk.MockKException: no answer found for: TripNotification(#4).getNotificationId()**

Reply:
Because that's not a `Unit` function :upside_down_face: 

We can leave it as is, I'm just personally not a fan of relaxing the returned values as I've seen it lead to inaccurate test results.